### PR TITLE
documentation edits

### PR DESCRIPTION
**Pull request by jasonlmfong**
Going through the docs and just making small changes to improve readability.

### CODE REVIEW

**Comment by Landixbtw** [comment]
Grammar police

File: docs/writing-fusing-functions.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ main = (if true 42 37)
 // so (true 42 37) will do the same thing.
 ```
 
-This is how a `Not` function that acts on this encoding can be defined
+This is how a `Not` function that acts on this encoding can be defined.
```
Comment (developedby) [inline docs/writing-fusing-functions.md:20]:
```suggestion
This is how a `Not` function that acts on this encoding can be defined:
```

**Review by developedby — CHANGES_REQUESTED** [review]
I think there a lot of places that refer to entry points as entrypoints haha

**Review by developedby — APPROVED** [review]

**Review by developedby — APPROVED** [review]

**Comment by developedby** [comment]
Looks like something changed in the tests earlier. I'll merge this after I fix it

**Comment by developedby** [comment]
I've moved to #415 to fix the merge and add some other similar things.

**Comment by jasonlmfong** [comment]
Sounds good thanks

**Pull request closed without merging**